tests/test_actions.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for the `edx-enterprise` admin actions module.
+"""
+from __future__ import absolute_import, unicode_literals
+
+import unittest
+
+import factory
+import mock
+
+from enterprise.actions import export_as_csv_action
+
+
+class DummyModel(object):
```

Comment:
It doesn't have to be an either/or here -- both models could be tested sequentially if we want to ensure that the workflow executes correctly for both EnterpriseCustomer and another model.